sbi/simulators/user_input_checks_utils.py

```diff
@@ -214,7 +239,13 @@ def log_prob(self, value) -> Tensor:
         # Sum accross last dimension to get joint log prob over all distributions.
         return torch.cat(log_probs, dim=1).sum(-1)
 
-    def _prepare_value(self, value):
+    def _prepare_value(self, value) -> Tensor:
```

Comment:
value is a bit generic but if you think of contributing this back to PyTorch, using `parameter`/`theta` is not a solution either. Hmm...

Reply:
is called it value because it is called value in `torch.distributions.Distribution.log_prob(value)`.